Git Formats/Git Diff.sublime-syntax

```diff
@@ -0,0 +1,511 @@
+%YAML 1.2
+---
+# https://www.sublimetext.com/docs/syntax.html
+# https://git-scm.com/docs/git-format-patch
+# https://git-scm.com/docs/diff-format
+name: Git Diff
+scope: source.diff.git
+version: 2
+extends: Packages/Diff/Diff.sublime-syntax
+
+file_extensions:
+  - diff.eml
+  - patch.eml
+  - diff.mbox
+  - patch.mbox
+
+first_line_match: |-
+  ^(?x:
+    diff[ ]--(?:git|combined|cc)[ ]
+    | From [ ] \w+@z [ ]
+      Thu [ ] Jan [ ]{2} 1 [ ] 00:00:00 [ ] 1970  # git-send-email
+    | From [ ] (?:[0-9a-fA-F]{40}|[0-9a-fA-F]{64}) [ ]
+      Mon [ ] Sep [ ]   17 [ ] 00:00:00 [ ] 2001  # git-format-patch
+  )
+
+variables:
+  hash: '[0-9a-fA-F]{7,64}'
+  day3: (?:Mon|Tue|Wed|Thu|Fri|Sat|Sun)
```

Comment:
Found https://github.com/git/git/tree/master/po, but I'm not seeing month and day-of-week abbreviations.

Reply:
FWIW, language specific patterns in _Git Commit_ syntax were added as those are translated to set UI language by git and wouldn't otherwise be highlighted in non-english environments.

AFAIK abbreviation in date-time strings are not translated.